content/en/api/monitors/code_snippets/api-monitor-delete.rb

```diff
@@ -7,4 +6,4 @@
 dog = Dogapi::Client.new(api_key, app_key)
 
 # Delete a monitor
-dog.delete_monitor(62625)
\ No newline at end of file
+dog.delete_monitor(62_625)
```

Comment:
If find this slightly confusing with `_` 
Is this a ruby way?

Reply:
Yeah, this is the idiomatic Ruby way for long numbers:

- https://docs.ruby-lang.org/en/2.0.0/syntax/literals_rdoc.html#label-Numbers
- https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/NumericLiterals

Reply:
I can disable it in rubocop if you want - and then we can run the autoformatter to remove the underscores 🙏

Reply:
no all good!